### PR TITLE
Deprecate Announcements

### PR DESCRIPTION
**Pull request by ct-martin**
This is a partial completion of #163

- Replace with CampusGroups in nav
- Remove from home & add message
- Merge /announcements/archive back into /announcements
- Add "New location TBA" to /announcements

### CODE REVIEW

File: _data/tabs.yml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 - name: Home
   link: /
 
-- name: Announcements
-  link: /announcements/
+#- name: Announcements
```
Comment (jwflory) [inline _data/tabs.yml]: I suggest leaving this line blank for readability. The history exists in version control so there's no need to keep it in the file.

```suggestion
```

File: _data/tabs.yml [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
+#  link: /announcements/
```
Comment (jwflory) [inline _data/tabs.yml]: I suggest leaving this line blank for readability. The history exists in version control so there's no need to keep it in the file.

```suggestion
```

**Comment by ct-martin** [comment]
@jwflory I left those commented to follow the way we've been doing things so far, and because we don't have a new solution in yet. I added a note in the relevant issue to do some cleanup once the new location is finalized.

**Comment by jwflory** [comment]
@ct-martin Works for me. It is a personal preference for me. Merging! :clapper:

**Pull request merged**